services/services.go

```diff
@@ -3,7 +3,9 @@ package services
 import (
 	// Importing the services so they register their services to SDA
 	// automatically when importing github.com/dedis/cothority/services
-	_ "github.com/dedis/cosi/service"
+	// XXX We should not import that doing weird cross dependancy between
+	// packages
+	//	_ "github.com/dedis/cosi/service"
```

Comment:
Not sure what this comment means. If it's legitimate to remove `_ "github.com/dedis/cosi/service"`, you can just do it, or not?